stellargraph/core/graph.py

```diff
@@ -237,6 +237,69 @@ def out_nodes(self, node: Any) -> Iterable[Any]:
         """
         raise NotImplementedError
 
+    def nodes_of_type(self, node_type=None):
+        """
+        Get the nodes of the graph with the specified node types.
+
+        Args:
+            node_type: a type of nodes that exist in the graph
```

Comment:
```suggestion
            node_type (str, optional): a type of nodes that exist in the graph
```
I think?

Reply:
I don't think it's that strict; at the moment, I think it can be any type (or maybe any hashable type), e.g. some of our tests use integers for the labels:

https://github.com/stellargraph/stellargraph/blob/bdb3a13cb171f1965ad5279b021b3587d0560e93/tests/core/test_stellargraph.py#L451-L452

Reply:
I've gone with

```suggestion
            node_type (hashable, optional): a type of nodes that exist in the graph
```

to match some other places that use this.